docs/src/declarative_database_management.md

```diff
@@ -1,54 +1,130 @@
-# Declarative Database Management
+# PostgreSQL Database Management
 
-Declarative database management enables users to control the lifecycle of
-databases via a new Custom Resource Definition (CRD) called `Database`.
+CloudNativePG simplifies PostgreSQL database provisioning by automatically
+creating an application database named `app` by default. This default behavior
+is explained in the ["Bootstrap an Empty Cluster"](bootstrap.md#bootstrap-an-empty-cluster-initdb)
+section.
 
-A `Database` object is managed by the instance manager of the cluster's
-primary instance. This feature is not supported in replica clusters,
-as replica clusters lack a primary instance to manage the `Database` object.
+For more advanced use cases, CloudNativePG introduces **declarative database
+management**, which empowers users to define and control the lifecycle of
+PostgreSQL databases using the `Database` Custom Resource Definition (CRD).
+This method seamlessly integrates with Kubernetes, providing a scalable,
+automated, and consistent approach to managing PostgreSQL databases.
 
-### Example: Simple Database Declaration
+---
 
-Below is an example of a basic `Database` configuration:
+## Key Concepts
+
+### Scope of Management
+
+!!! Important
+    CloudNativePG manages **global objects** in PostgreSQL clusters, such as
+    databases, roles, and tablespaces. However, it does **not** manage the content
+    of databases (e.g., schemas and tables). For database content, specialized
+    tools or the applications themselves should be used.
```

Comment:
Not sure about "application themselves". Which applications? I can make that you're talking about the user payloads, but IMO this is not obvious.
```suggestion
    tools, or the user-defined payloads should be used.
```